chapters/io/index.rst

```diff
@@ -0,0 +1,315 @@
+How to handle input and output for Flows
+=======================================
+
+This chapter explains how to provide input data to the `Flow API`. Moreover, it describes how to handle the output via
+callback functions.
+
+Input
+-----
+The input data for the Flow operations `Flow.index`, `Flow.update` and `Flow.search` can be provided in three different ways.
+In the following, the input functionality is shown using the `Flow.index` function. `Flow.update` and `Flow.search` are used analogous.
```

Comment:
```suggestion
In the following, the input functionality is shown using the `Flow.index` function. `Flow.update` and `Flow.search` are similar.
```

Reply:
Changed it to:
For `Flow.update` and `Flow.search`, the input is provided the same way.